packages/create-toolpad-app/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "create-toolpad-app",
+  "version": "0.0.42-alpha.5",
+  "keywords": [
+    "react",
+    "toolpad",
+    "mui"
+  ],
+  "description": "Create Toolpad apps with one command",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mui/mui-toolpad",
+    "directory": "packages/create-toolpad-app"
+  },
+  "author": "Toolpad Team",
+  "license": "MIT",
+  "bin": {
+    "create-toolpad-app-test": "./dist/index.js"
+  },
+  "files": [
+    "dist"
+  ],
+  "type": "module",
+  "scripts": {
+    "dev13": "tsc -w -p ./tsconfig.json --outDir ./dist/",
+    "prerelease": "rimraf ./dist/",
```

Comment:
```json
    "prebuild": "rimraf ./dist/",
    "build": "tsc -p ./tsconfig.json -outDir ./dist/",
```

Let's use the same script names as in `@mui/toolpad-core` and `@mui/toolpad-components`?